bem-css/style.css

```diff
@@ -76,7 +76,7 @@ body {
   color: #FFF;
 }
 
-.label__field {
+.input {
   display: block;
```

Comment:
Fixed.

Added proposal for layout separation in second commit: https://github.com/AleshaOleg/holy-grail-markup/pull/17/commits/6fbc90d9a073b9f3f6f6f02fc3f5f58deac251ce